transformer/src/main/scala/uk/ac/wellcome/transformer/parsers/TransformableParser.scala

```diff
@@ -0,0 +1,29 @@
+package uk.ac.wellcome.transformer.parsers
+
+import com.gu.scanamo.error.DynamoReadError
+import com.twitter.inject.Logging
+import uk.ac.wellcome.models.Transformable
+import uk.ac.wellcome.transformer.receive.RecordMap
+
+import scala.util.Try
+
+trait TransformableParser[+T <: Transformable] extends Logging {
```

Comment:
I have briefly and badly explained, but here are a few references as to what that means: http://docs.scala-lang.org/tutorials/tour/variances.html and http://docs.scala-lang.org/tutorials/tour/upper-type-bounds.html